packages/ui/package.json

```
@@ -23,7 +23,8 @@
     "@littlespoon/link": "1.2.1",
     "@littlespoon/tag": "1.3.1",
     "@littlespoon/theme": "1.13.0",
-    "@littlespoon/typography": "1.5.0"
+    "@littlespoon/typography": "1.5.0",
+    "label": "1.0.0"
```

This should be `@littlespoon/label`:

```suggestion
    "@littlespoon/label": "1.0.0"
```

---

Updated